modules/manage/pages/audit-logging.adoc

```diff
@@ -0,0 +1,130 @@
+
+Even though audited event messages are stored to a specialized immutable topic, standard topic settings still apply. For example, you can apply the same tiered storage, retention time, and replication settings available to normal topics. These particular options are important for controlling the amount of disk space utilized by your audit topics.
+
+IMPORTANT: You must configure certain audit logging properties before enabling audit logging because these settings impact the creation of the `__audit_log` topic itself. These properties include: `audit_log_num_partitions` and `audit_log_replication_factor`. The Kafka API allows you to add partitions or alter the replication factor after enabling audit logging, but Redpanda prevents you from altering these two configuration values directly.
+
+== Audit logging event types
+
+Redpanda's auditable events fall into one of eight different event types. The APIs associated with each event type are as follows.
+
+[cols="1,3"]
+|===
+|Audit event type |Associated APIs
+
+|management
+a|* AlterPartitionReassignments
+* CreateACLs
+* CreatePartitions
+* CreateTopics
+* DeleteACLs
+* DeleteGroups
+* DeleteRecords
+* DeleteTopics
+* IncrementalALterconfigs
+* OffsetDelete
+
+|produce
+a|* AddPartitionsToTxn
+* EndTxn
+* InitProducerId
+* Produce
+
+|consume
+a|* AddOffsetsToTxn
+* Fetch
+* JoinGroup
+* LeaveGroup
+* ListOffset
+* OffsetCommit
+* SyncGroup
+* TxOffsetCommit
+
+|describe
+a|* DescribeACls
+* DescribeConfigs
+* DescribeGroups
+* DescribeLogDirs
+* FindCoordinator
+* ListGroups
+* ListPartitionReassginments
+* Metadata
+* OffsetForLeaderEpoch
+* DescribeProducers
+* DescribeTransations
+* ListTransactions
+
+|heartbeat
+a|* Heartbeat
+
+|authenticate
+a|* All authentication events
+
+|schema_registry
+a|* All Schema Registry API calls
+
+|admin
+a|* All Admin API calls
+|===
+
+
+== Enabling audit logging
+
+All audit log settings are applied at the cluster level using rpk. Use the `rpk cluster config` to configure audit logs. Some options will require a cluster restart. You can confirm this using `rpk cluster config status`.
+
+Some key steps to tuning your audit logging settings include:
+
+* If you wish to change the number of partitions or the replication factor for your audit log topic, set the `audit_log_num_partitions` and `audit_log_replication_factor` properties respectively.
+* Choose the type of events needed by setting `audit_enabled_event_types` to the desired list of event categories. Keep this as restrictive as possible based on your compliance and security needs to avoid excessive noise in your audit logs.
+* Identify non-sensitive topics so that you can exclude them from auditing. Set `audit_exclude_topics` to this list of topics.
+* Identify non-sensitive principles so that you can exclude them from auditing. Set `audit_exclude_principles` to this list of principles. This command accepts names in the form of `name` or `User:name`.
```

Comment:
```suggestion
* Identify non-sensitive principles so that you can exclude them from auditing. Specify this list of principals in  `audit_exclude_principles`. This command accepts names in the form of `name` or `User:name`.
```